integration/src/test/java/edu/unc/lib/boxc/integration/web/access/CollectionsEndpointIT.java

```diff
@@ -101,23 +48,14 @@ public void setup() throws Exception {
 
     @Test
     public void testCollectionsJsonOnlyReturnsAdminUnits() throws Exception {
-        var adminUnit1 = adminUnitFactory.createAdminUnit(Map.of("title", "Object1"));
-        var adminUnit2 = adminUnitFactory.createAdminUnit(Map.of("title", "Object2"));
-        var collection = collectionFactory.createCollection(adminUnit1, Map.of("title", "Object" + System.nanoTime()));
-        var work = workFactory.createWork(collection, Map.of("title", "Object" + System.nanoTime()));
-        var fileOptions = Map.of(
-                "title", "Object" + System.nanoTime(),
-                WorkFactory.PRIMARY_OBJECT_KEY, "false",
-                FileFactory.FILE_FORMAT_OPTION, FileFactory.AUDIO_FORMAT);
-        workFactory.createFileInWork(work, fileOptions);
-        folderFactory.createFolder(collection, Map.of("title", "Object" + System.nanoTime()));
+        createDefaultObjects();
 
         try (var resp = httpClient.execute(getMethod)) {
             var metadata = getMetadataFromResponse(resp);
 
-            assertSuccessfulResponse(resp);
-            assertValuePresent(metadata, 0, "type", "AdminUnit");
-            assertValuePresent(metadata, 1, "type", "AdminUnit");
+            SearchEndpointTestUtility.assertSuccessfulResponse(resp);
```

Comment:
Definitely don't need to change this, just wanted to mention that there is the option to do a `import static` of these methods, similar to how the assertEquals get imported. While it can improve readability, it can also make debugging harder since it's a little more difficult to know where the methods are coming from.